setup.py

```diff
@@ -81,12 +81,15 @@ def load_deps(path):
     # existing (as of writing this)
     "s3": [
         "boto3~=1.9, <1.17.50",
-        "botocore>=1.20.49,<1.20.50",
-        "aiobotocore<=1.3.1",
-        "s3fs>=0.5.0, <=0.6.0",
+        "botocore>=1.20.106,<1.20.107",
+        "aiobotocore~=1.4.0",
+        "s3fs>=0.5.0, <=2021.8.1",
     ],
     # <12.7.0 from adlfs 0.6.3
-    "azure-blob-storage": ["azure-storage-blob~=12.0, <12.7.0", "adlfs~=0.7.1"],
+    "azure-blob-storage": [
+        "azure-storage-blob~=12.0, <12.7.0",
+        "adlfs>=0.7.1, <=2021.8.1",
+    ],
```

Comment:
Update the comments above the requirements (explaining the reasons for the different bounds)